webpack/assets/javascripts/react_app/components/FactCharts/FactChartConstants.js

```diff
@@ -0,0 +1,5 @@
+export const FACT_CHART_DATA_REQUEST = 'FACT_CHART_DATA_REQUEST';
+export const FACT_CHART_DATA_SUCCESS = 'FACT_CHART_DATA_SUCCESS';
+export const FACT_CHART_DATA_FAILURE = 'FACT_CHART_DATA_FAILURE';
+export const OPEN_FACT_CHART_MODAL = 'OPEN_FACT_CHART_MODAL';
+export const CLOSE_FACT_CHART_MODAL = 'CLOSE_FACT_CHART_MODAL';
```

Comment:
Could we rename to `FACT_CHART_MODAL_OPEN/CLOSE`

Reply:
Changed